src/govuk/components/checkboxes/checkboxes.js

```diff
@@ -99,10 +99,9 @@ Checkboxes.prototype.unCheckAllInputsExcept = function ($input) {
     var hasSameFormOwner = ($input.form === $inputWithSameName.form)
     if (hasSameFormOwner && $inputWithSameName !== $input) {
       $inputWithSameName.checked = false
+      this.syncConditionalRevealWithInputState($inputWithSameName)
     }
-  })
-
-  this.syncAllConditionalReveals()
+  }.bind(this))
```

Comment:
I don't really understand why this is necessary... is this a stylistic thing or did something break without it?

Reply:
We're now referencing `this` from within a nested function (on line 102).

By default, [`this` in the context of a function will be undefined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this#function_context). The call to `bind` allows us to keep the context of the outer function so that `this` resolves correctly.

Once upon a time, this is the sort of thing we used to work around by aliasing `this` in the outer scope…

```js
  var that = this;

  var allInputsWithSameName = document.querySelectorAll('input[type="checkbox"][name="' + $input.name + '"]')

  nodeListForEach(allInputsWithSameName, function ($inputWithSameName) {
    var hasSameFormOwner = ($input.form === $inputWithSameName.form)
    if (hasSameFormOwner && $inputWithSameName !== $input) {
      $inputWithSameName.checked = false
      that.syncConditionalRevealWithInputState($inputWithSameName)
    }
  })
```

Reply:
Ah, I understand now, thanks!